packages/presentation/src/components/markup/NodeContent.svelte

```diff
@@ -73,7 +74,26 @@
       {/each}
     {/if}
   {:else if node.type === MarkupNodeType.text}
-    {node.text}
+    {#await getResource(emojiPlugin.functions.ParseTextWithEmojis) then parseFunction}
```

Comment:
I'm not happy with it, Lets make a non async -> callback function, store parsedText into a local variable, remember result of parseFunction and reuse-it, so in must case it will not be async call at all.